### ISSUE TITLE
Add button to open the issue or PR or thread from a summary

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode-pull-request-github#6397]
Currently after summarizing an issue/PR/notification thread, we add a button that says `Mark as Read` at the end. It would be nice to have another button to open the issue/PR/notification thread after having read the summary. I imagine there may be people who may want to see the original.

cc @alexr00 @lszomoru

**Comment by lszomoru** [comment]
Absolutely. We have have "Mark as Read", "Mark as Done", and "Open" actions.
